app/controllers/settings.js

```diff
@@ -0,0 +1,14 @@
+import Ember from 'ember';
+
+const { Controller, get, inject, computed } = Ember;
+
+export default Controller.extend({
+  session: inject.service(),
+  isAuthenticated: computed.oneWay('session.isAuthenticated'),
+
+  actions: {
+    signOut() {
```

Comment:
Even better is to use the `route-action` addon and use it like a closure action.

https://github.com/DockYard/ember-route-action-helper